src/Crypto/crypto-pairings-test.asd

```diff
@@ -4,7 +4,7 @@
                core-crypto
                crypto-pairings)
   :perform  (test-op (o s)
-                     (symbol-call :lisp-unit :run-tests
+                     (symbol-call :lisp-unit :run-test
```

Comment:
@dbmcclain Actually, that needs to be `:run-tests` not `:run-test`, as thats the API of `lisp-unit` which actually runs the tests.